src/Tizen.NUI.Components/Controls/RecyclerView/LinearLayoutManager.cs

```diff
@@ -74,47 +118,36 @@ public override void Layout(float scrollPosition)
         public override List<RecycleItem> Recycle(float scrollPosition)
         {
             List<RecycleItem> result = new List<RecycleItem>();
-
-            float itemSize = LayoutOrientation == Orientation.Horizontal?ItemSize.Width:ItemSize.Height;
-
-            View firstVisibleItem = Container.Children[3];
-            float firstVisibleItemPosition = LayoutOrientation == Orientation.Horizontal?firstVisibleItem.Position.X:firstVisibleItem.Position.Y;
-
-            firstVisibleItemPosition += 180;
-
+          
             bool checkFront = (mPrevScrollPosition - scrollPosition) > 0;
 
             if(checkFront)
             {
-                if(firstVisibleItemPosition < Math.Abs(scrollPosition))
+                if(firstVisibleItemIndex > 3)
```

Comment:
what is the meaning of 3?

Reply:
I want to keep only two items at tip of each side.
So if index is over 2, recycle item.

It should be set by property later.